src/main/java/net/atos/zac/notificaties/NotificatieReceiver.java

```diff
@@ -120,6 +116,15 @@ private boolean isAuthenticated(final HttpHeaders headers) {
         return secret.equals(headers.getHeaderString(HttpHeaders.AUTHORIZATION));
     }
 
+    private void handleProductaanvraag(final Notificatie notificatie) {
+        final String objecttypeUri = notificatie.getProperties().get(OBJECTTYPE_KENMERK);
+        // only attempt to handle productaanvraag if the notification resource is an object with 'CREATE' action
+        // and has an object type defined
+        if (notificatie.getResource() == OBJECT && notificatie.getAction() == CREATE && !isEmpty(objecttypeUri)) {
+            productaanvraagService.handleProductaanvraag(uuidFromURI(notificatie.getResourceUrl()));
+        }
```

Comment:
Shouldn't we log a warning that we don't handle the notification

Reply:
No because it is not an exceptional case at all. The whole handling of notifications is a fall-through mechanism.